crates/re_types_builder/src/objects.rs

```diff
@@ -607,6 +610,11 @@ impl Object {
             .is_some()
     }
 
+    /// Is plain-old-data (i.e. no destructor)?
+    pub fn is_pod(&self, objects: &Objects) -> bool {
```

Comment:
Feel free to rename it as you see fit. I only use it to produce slightly nicer code in some cases. I think it is worth spending a small amount of effort making the generated code nice and easy to read, since the source code will for many act as documentation.